### ISSUE TITLE
Bad password strength metric

### ISSUE THREAD

**Issue by jkopczyn** [linuxmint/cinnamon#10368]
```
 * Cinnamon version 5.0.5
 * Distribution - Mint 20.2
 * 64 bit
 ```

**Issue**
Password change dialog has faulty strength measurement. A six-word diceware password displays as 'weak'.

**Steps to reproduce**
Change password in the system dialog, enter a strong password using the diceware/'correct horse battery staple' style.

**Expected behaviour**
A four to five word diceware password should be marked as medium strength and a six-plus word password should be marked strong.

**Comment by jkopczyn** [comment]
Behavior unchanged as of now.

**Comment by JosephMcc** [comment]
Looks like we use the same code that Ubiquity uses to determine password strength: https://github.com/linuxmint/cinnamon/blob/master/files/usr/share/cinnamon/cinnamon-settings/modules/cs_user.py#L424

That makes sense so it will match what you get when you do the install. This will also be handled in a separate app going forward in Mint. Closing this since it isn't really a bug.

**Comment by jkopczyn** [comment]
No, that's still a bug. You can call it a wontfix if you like but it's a bug.
